### PR TITLE
Remove useless __forceinline and forceinline macros

### PR DESCRIPTION
**Pull request by jsakkine**
These macros look fairly useless so just replace the uses with the
"regular" inline.

Signed-off-by: Jarkko Sakkinen <jarkko.sakkinen@iki.fi>

### CODE REVIEW

**Comment by jsakkine** [comment]
Came up while doing cmake build. Does make much sense to duplicate those definitions...

File: src/common/precompiled.h [hunk 1]
```diff
@@ -4,5 +4,5 @@
 
 #include "globals.h"
 
-#define forceinline __forceinline
+#define inline inline
```
Comment (whydoubt) [inline src/common/precompiled.h]: Isn't this is redundant?
The line should probably just be removed.

Comment (jsakkine) [reply src/common/precompiled.h]: It is. I made this commit by

```
$ find -regex ".*\.cpp\|.*\.h"|xargs sed -i 's/__forceinline/inline/'
$ find -regex ".*\.cpp\|.*\.h"|xargs sed -i 's/forceinline/inline/'
```
Thanks for catching that one up. Will update.

**Comment by jsakkine** [comment]
Updated.

File: src/common/precompiled.h (hunk 1 above)
Comment (baconpaul) [reply src/common/precompiled.h]: Might be reading diffs wrong, but it looks like you got this in vstquartzplug.h but not in precompiled.h.

**Comment by baconpaul** [comment]
I confirmed your master builds AU and VST3 mac both with these changes, and @whydoubt and I both reviewed, so I am going to merge them. I'll rebase my logging branch after that.

**Pull request merged**